tests/tests/test_conf.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+from pipeline.conf import PipelineSettings
+
+class TestSettings(TestCase):
+
+    def test_3unicode(self):
+        s = PipelineSettings(dict(), DEFAULTS={ "PIPELINE_FOO_BINARY": "env actualprogram" })
```

Comment:
Hrm, I'm not sure how to use that. The settings that django.test.utils.override_settings retrieves to update doesn't have the contents of pipeline.conf.settings , right? If it's right to use o_s(), give me a hint?